Specs/Cesium3DTilesSpecHelper.js

```diff
@@ -0,0 +1,262 @@
+/*global define*/
+define([
+        'Core/Color',
+        'Core/defaultValue',
+        'Core/OrientedBoundingBox',
+        'Scene/Cesium3DTileContentProviderFactory',
+        'Scene/Cesium3DTileContentState',
+        'Scene/Cesium3DTileset',
+        'Specs/pollToPromise'
+    ], function(
+        Color,
+        defaultValue,
+        OrientedBoundingBox,
+        Cesium3DTileContentProviderFactory,
+        Cesium3DTileContentState,
+        Cesium3DTileset,
+        pollToPromise) {
+    "use strict";
+
+    var Cesium3DTilesSpecHelper = function() {
```

Comment:
We avoid using `Helper` in names.  Do you have a better name?  I dunno, `Cesium3DTilesTester`.